src/typescript.ts

```diff
@@ -0,0 +1,6 @@
+import * as ts from 'typescript/lib/typescript'
+import {patchCreateProgram} from './patchCreateProgram'
+
+patchCreateProgram(ts)
+
+export default ts
```

Comment:
Original 'typescript/lib/typescript' returns  `module.exports = ts`, but you return `module.exports.default = ts`

Reply:
Use instead `export = ts;`

Reply:
Or do it more functional `export = patchCreateProgram(ts)`